<launch>

<node pkg="truenorth" type="gyro_publisher" name="gyro_publisher" output="screen" respawn="true">

      	       <param name="rate" value="10" type="int" />
	       <param name="hz" value="5000" type="int" />
	       <param name="port" value="/dev/ttyUSB1" type="str" />
	       <param name="baud" value="4147200" type="int" />
	       <param name="lat" value="32.4" type="double" />
	       <param name="r_align" value="[-1.57080,0,1.57080]" type="string" />
	       <param name="r0" value="[0,0,0]" type="string" />
	       <param name="ang_bias" value="[0,0,0]" type="string" />
	       <param name="acc_bias" value="[0,0,0]" type="string" />
	       <param name="k_g" value="[0.01,0.01,0.01]" type="string" />
	       <param name="k_north" value="[0.1,0.1,0.1]" type="string"/>
	       <param name="k_acc" value="[0.1,0.1,0.1]" type="string"/>
	       <param name="k_acc_bias" value="[0.5,0.5,0.5]" type="string" />
	       <param name="k_ang_bias" value="[0.0000025,0.0000025,0.0000025]" type="string" />
	       <param name="k_E_n" value="[0.000025,0.000025,0.000025]" type="string" />
	       
        </node> 


</launch>
